lib/admin-page.php

```diff
@@ -607,8 +607,10 @@ function classicpress_check_can_migrate() {
 	}
 
 	// Check: Supported PHP version
-	$php_version_min = '5.6';
-	if ( version_compare( PHP_VERSION, $php_version_min, 'lt' ) ) {
+	if ( isset( $parameters['php'] ) &&
+		version_compare( PHP_VERSION, $parameters['php']['min'], 'lt' ) ||
+		version_compare( PHP_VERSION, $parameters['php']['unsupported'], 'ge' )
```

Comment:
This says `unsupported` with a `>= / ge` comparison. Below, the parameter is called `max` and the text says `version %1 to %2` which is consistent with a `max` version. I think the best way to do this is to use `max` and a `> / gt` comparison.

Reply:
I did play with that initially but the issue we face is with the semver versioning and checks.

The endpoint states maximum supported PHP is 7.4.x, if we change that to 7.4.9999 for example it doesn't look great to users who then question if that version exists.

Perhaps we could do some string substitution and alter 7.4.999 to 7.4.x for user readability.

Reply:
API endpoint PR updated and this PR now amended as above as a result of changes there.